doc/installation.rst

```diff
@@ -5,11 +5,29 @@ Installing CadQuery
 
 CadQuery 2 is based on
 `OCP <https://github.com/CadQuery/OCP>`_,
-which is a set of Python bindings for the open-source `OpenCascade <http://www.opencascade.com/>`_ modelling kernel.
+which is a set of Python bindings for the open-source ` OpenCascade <http://www.opencascade.com/>`_ modelling kernel.
```

Comment:
Why the space in front of "OpenCascade" here?

Reply:
Typo, fixed